Zavala/AppDelegate.swift

```diff
@@ -629,47 +463,47 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		builder.remove(menu: .openRecent)
 		
 		// Application Menu
-		let appMenu = UIMenu(title: "", options: .displayInline, children: [showPreferences, checkForUpdates])
+        let appMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.showPreferences, checkForUpdates])
 		builder.insertSibling(appMenu, afterMenu: .about)
 		
 		// File Menu
-		let syncMenu = UIMenu(title: "", options: .displayInline, children: [sync])
+        let syncMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.sync])
 		builder.insertChild(syncMenu, atStartOfMenu: .file)
 
-		let importExportMenu = UIMenu(title: "", options: .displayInline, children: [importOPMLCommand, exportMarkdownCommand, exportOPMLCommand])
+        let importExportMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.importOPMLCommand, UIKeyCommand.exportMarkdownCommand, UIKeyCommand.exportOPMLCommand])
 		builder.insertChild(importExportMenu, atStartOfMenu: .file)
 
-		let newWindowMenu = UIMenu(title: "", options: .displayInline, children: [newWindowCommand])
+        let newWindowMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.newWindowCommand])
 		builder.insertChild(newWindowMenu, atStartOfMenu: .file)
 
-		let newItemsMenu = UIMenu(title: "", options: .displayInline, children: [newOutlineCommand])
+        let newItemsMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.newOutlineCommand])
 		builder.insertChild(newItemsMenu, atStartOfMenu: .file)
 
-		let shareMenu = UIMenu(title: "", options: .displayInline, children: [sendCopyCommand])
+        let shareMenu = UIMenu(title: "", options: .displayInline, children: [sendCopyCommand])
 		builder.insertChild(shareMenu, atEndOfMenu: .file)
 
 		// Edit
-		let linkMenu = UIMenu(title: "", options: .displayInline, children: [linkCommand])
+        let linkMenu = UIMenu(title: "", options: .displayInline, children: [UIKeyCommand.linkCommand])
 		builder.insertSibling(linkMenu, afterMenu: .standardEdit)
 
 		// Format
 		builder.remove(menu: .format)
-		let formatMenu = UIMenu(title: L10n.format, children: [toggleBoldCommand, toggleItalicsCommand])
+        let formatMenu = UIMenu(title: L10n.format, children: [UIKeyCommand.toggleBoldCommand, UIKeyCommand.toggleItalicsCommand])
 		builder.insertSibling(formatMenu, afterMenu: .edit)
 
-		// View Menu
-		let expandCollapseMenu = UIMenu(title: "",
-										options: .displayInline,
-										children: [expandAllInOutlineCommand, expandAllCommand, expandCommand, collapseAllInOutlineCommand, collapseAllCommand, collapseCommand])
-		builder.insertChild(expandCollapseMenu, atStartOfMenu: .view)
-		let toggleFilterOutlineMenu = UIMenu(title: "", options: .displayInline, children: [toggleOutlineFilterCommand, toggleOutlineHideNotesCommand])
+        // View Menu
```

Comment:
When this happens it means that your indentation is different than that of the project. Make sure it's the same (tabs or spaces). Once you fix it, these changes should mostly disappear.